react-performance/src/components/CountryTable.tsx

```diff
@@ -0,0 +1,314 @@
+import { useEffect, useState, useMemo, Fragment } from 'react';
+import { useSuspenseQuery } from '@tanstack/react-query';
+
+interface CountyInfo {
+  year: number;
+  population: number;
+  cement_co2: number;
+  cumulative_cement_co2: number;
+  cumulative_luc_co2: number;
+  land_use_change_co2: number;
+  land_use_change_co2_per_capita: number;
+  methane: number;
+  methane_per_capita: number;
+}
+
+interface CountryRaw {
+  [key: string]: {
+    iso_code: string;
+    data: CountyInfo[];
+  };
+}
+
+interface CountryItem {
+  name: string;
+  iso_code: string;
+  data: CountyInfo[];
+}
```

Comment:
would be great to clean up this file and separate types, defaults, util functions into its own files